build.gradle

```diff
@@ -7,12 +7,12 @@ plugins {
 apply plugin: "kotlin"
 
 group 'dev.blachut.svelte.lang'
-version '0.12.1'
+version '0.12.2'
```

Comment:
please don't bump version outside of release commits (I should've mentioned it in README)